components/UserList.js

```diff
@@ -73,6 +73,18 @@ function UserList ({ type }) {
   const result = useSWR(url)
   const { data, error } = result
   const users = data && data.data
+  if (summaryView) {
+    const total = data ? data.total : 0
+    return (
+      <div style={{display: 'inline-block', margin: '10px', textAlign: 'center'}}>
+        <div style={{fontSize: 'xxx-large'}}>{total}</div>
+        <div>{selectedType.label}</div>
+        <Button onClick={onViewUser} size='sm' variant='outline-primary'>
+          View
+        </Button>
+      </div>
```

Comment:
My personal preference would be for the whole block element to be the button.

Reply:
I like the idea of that, but don't want us to spend too much time on intricate styling this project (since it's more of an administrative focus). Let's hold off for now.